spec/fabricators/result_link_fabricator.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+Fabricator(:result_link) do
+  link_type { FFaker.rand 3 }
```

Comment:
We should better fabricate it from the model enum 🙌 
```suggestion
  link_type { ResultLink.link_types.keys.sample }
```

Reply:
Fixed d89551c2baf0587728497bdc2cfe9ab94c55db10